test/e2e/run-e2e.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+script_path="$(dirname -- "${BASH_SOURCE[0]}")"
+
+main() {
+    echo "Running E2E tests" >&2
+    # shellcheck disable=SC1091
+    source "${script_path}/vars.sh"
+    local github_org="${GH_ORG:?A GitHub org where the https://github.com/redhat-appstudio-qe/konflux-ci-sample repo is present/forked should be provided}"
+    local github_token="${GH_TOKEN:?A GitHub token should be provided}"
+    local quay_dockerconfig="${QUAY_DOCKERCONFIGJSON:?quay.io credentials in .dockerconfig format should be provided}"
+    docker run --network=host -v ~/.kube/config:/kube/config --env KUBECONFIG=/kube/config -e GITHUB_TOKEN="$github_token" -e QUAY_TOKEN="$(base64 <<< "$quay_dockerconfig")" -e MY_GITHUB_ORG="$github_org" -e E2E_APPLICATIONS_NAMESPACE=user-ns2 "$E2E_TEST_IMAGE" "--ginkgo.label-filter=upstream-konflux" "--ginkgo.focus=Test local" "--ginkgo.v"
```

Comment:
I think that we should check if podman is installed, if so use it. Will help to support users that want to run e2e test locally and use podman.